Develop

AndreaFan123 opened the pull request:
Discord Name : AndreaF

@cwpeng  老師好：以下是我的 api 驗證網址，請老師過目

- 頁數：http://44.198.225.149:3000/api/attractions?page=0

- 超過頁面顯示錯誤訊息：http://44.198.225.149:3000/api/attractions?page=7

- 關鍵字：http://44.198.225.149:3000/api/attractions?page=0&keyword=%E6%BA%AB%E6%B3%89

- 該頁面無結果：http://44.198.225.149:3000/api/attractions?page=4&keyword=%E6%BA%AB%E6%B3%89

- ID 查詢頁面：http://44.198.225.149:3000/api/attraction/3

cwpeng commented:
1. use "nextPage", not "NextPage" based on API spec.
2. "nextPage" of http://44.198.225.149:3000/api/attractions?page=4 and http://44.198.225.149:3000/api/attractions?page=0&keyword=%E6%BA%AB%E6%B3%89 should be null because they have reached final pages of results.

Fix issues, leave comment for me to review again.

AndreaFan123 commented:
Hello @cwpeng ,

I've updated the code, kindly check.

1. Fixed title from NextPage to nextPage
2. Fixed the issue of not showing null to the latest page. 

http://44.198.225.149:3000/api/attractions?page=0&keyword=%E6%BA%AB%E6%B3%89
http://44.198.225.149:3000/api/attractions?page=4

cwpeng reviewed (approved):
OK. Done.
